### DISCUSSION TITLE
Window Switcher - Move title from each application into a single title, similar to windows 7.

### DISCUSSION THREAD

**Discussion by madcdevelop** [valinet/ExplorerPatcher#1162] [Ideas]
Hello,

I love using the Window Switcher. I do like to keep the row height at 160 so it's small and I find it quicker to switch between apps as my eyes don't need to move that much. The issue with that is if the window is half the size or what not then the text of the application name is cut-off which makes it difficulte to know what the app is.
Is it possible to add a feature where the text for each application shows up in a single place above/below all the windows? This would be similar to how Windows 7 window switcher behaves.
I would use the Windows NT switcher but I do like using the feature that ignores windows/apps on the second monitor which helps declutter the space when switching windows/apps.

![image](https://user-images.githubusercontent.com/40719351/226692642-df20d70a-f8ef-4d2d-ba68-e43a83e1d48a.png)

**Comment by XuanChen1234** [comment]
You can ask from @valinet and @Amrsatrio
